spec/features/avo/password_spec.rb

```diff
@@ -58,6 +58,12 @@
         expect(page).to have_field type: "password", id: "user_password_confirmation", placeholder: "Password confirmation", text: nil
       end
 
+      it "checks password reveal button" do
+        visit url
+
+        expect(page).to have_selector "[data-password-visibility-target]"
+      end
```

Comment:
Besides this test, we need a system test where we test the action of showing/hiding the password